src/internal/model/world/level.go

```diff
@@ -37,17 +37,18 @@ type CollisionType int
 
 const (
 	CollisionTypeNone CollisionType = iota
-	CollisionTypeBorder
 	CollisionTypeEnemy
 	CollisionTypeItem
 	CollisionTypeTeleport
 )
 
 func NewLevelWithDefaults(random utils.Randomizer, mapSize primitives.Size2D[uint]) *Level {
 	cfg := DefaultLevelConfig(mapSize)
+	levelNumber := 1
 	return NewLevelWithComponents(
 		random,
 		cfg,
+		uint(levelNumber),
```

Comment:
Зачем отдельная переменная, когда можно просто указать значение? 1 - это допустимо и не является "магическим числом".

```suggestion
return NewLevelWithComponents(
    random,
    cfg,
		1U,
```